### ISSUE TITLE
Add an optional salt for the hash

### ISSUE THREAD

**Issue by mrmowgli** [tmeasday/meteor-crypto-md5#2]
To avoid hitting predictable hashes, please let us pass in a salt.

**Comment by tmeasday** [comment]
Send a PR? I'm not actually using this package right now. Happy to pass over control of it.

**Comment by mrmowgli** [comment]
I realize this is more appropriate for crypto-base, this is literally only the encryption functions, not the actual front end.  I was thinking in terms of a simple salt in line with http://skfox.com/2007/12/18/md5-hashes-and-salt/, 
so adding another signature:

```
CryptoJS.MD5('some-string', 'some-salt').toString()
```

with a fallback to the original if 'some-salt' was empty.  Has this file had any major changes from the google code?

**Comment by tmeasday** [comment]
No, basically none, apart from changing the way it declares `CryptoJS`.
